app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/state/MessageCompositionInputStateHolder.kt

```diff
@@ -139,66 +123,56 @@ class MessageCompositionInputStateHolder(val messageTextState: TextFieldState) {
         isTextExpanded = false
     }
 
-    fun clearFocus() {
-        inputFocused = false
-    }
-
-    fun requestFocus() {
+    fun setFocused() {
         inputFocused = true
+        keyboardController?.show()
     }
 
-    fun showOptions() {
-        optionsVisible = true
-        subOptionsVisible = true
-        if (initialKeyboardHeight > 0.dp) {
-            optionsHeight = initialKeyboardHeight
-        } else {
-            optionsHeight = keyboardHeight
+    fun requestFocus() {
+        if (!inputFocused) {
+            focusRequester.requestFocus()
+            focusRequester.captureFocus() // TODO check
```

Comment:
👀

Reply:
I will check and remove it after playtest